napari/layers/points/_points_mouse_bindings.py

```diff
@@ -82,19 +82,24 @@ def select(layer, event):
     layer._set_highlight(force=True)
 
 
+DRAG_THRESHOLD = 5
+
+
 def add(layer, event):
     """Add a new point at the clicked position."""
     # on press
```

Comment:
how about getting the `start_pos = event.pos` here and then below doing

```python
d = np.linalg.norm(event.pos - start_pos)
```

(which is position in screen pixels).  I had been thinking we'd drop `event.trail` in #2249.